doc/authentication.md

```diff
@@ -132,22 +132,24 @@ string response if there is no available key.
 
 Client certificate authentication
 ---------------------------------
-When a machine is joined to an Identity Management domain (like
-[FreeIPA](https://www.freeipa.org) or Active Directory) which has [client-side
-user certificates](https://www.freeipa.org/page/V4/User_Certificates) set up,
-then these can be enabled for authentication to Cockpit by setting this option
-in cockpit.conf:
 
+On a machine that has [sssd](https://sssd.io/) configured for mapping
+certificates to users, Cockpit can enable certificate based logins by setting
+these options in cockpit.conf:
 ```
 [WebService]
 ClientCertAuthentication = yes
+ClientCertAuthenticationCA = /path/to/ca.pem
```

Comment:
If we're allowing the client to present a certificate for login and we accept it as long as it's been signed (ie: no explicit list of valid certificates, server-side), then we absolutely also need to also support revocation.